internal/action/show_test.go

```diff
@@ -54,21 +62,15 @@ func TestShowMulti(t *testing.T) { //nolint:paralleltest
 	})
 
 	t.Run("show dir", func(t *testing.T) { //nolint:paralleltest
-		// first add another entry in a subdir
-		sec := secrets.NewKV()
-		sec.SetPassword("123")
-		assert.NoError(t, sec.Set("bar", "zab"))
-		assert.NoError(t, act.Store.Set(ctx, "bar/baz", sec))
-		buf.Reset()
-
 		c := gptest.CliCtx(ctx, t, "bar")
 		assert.NoError(t, act.Show(c))
 		assert.Equal(t, "bar/\n└── baz\n\n", buf.String())
 		buf.Reset()
 	})
 
+	require.NoError(t, act.cfg.Set("", "core.showsafecontent", "true"))
```

Comment:
Any reason to use a "global" one here instead of the explicit declaration in each test like it's done later in the same file for parsing?

Reply:
No. Not really ...